lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/util/ExceptionsTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.base.util;
+
+import io.trino.spi.ErrorCodeSupplier;
+import io.trino.spi.TrinoException;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static io.trino.plugin.base.util.Exceptions.findErrorCode;
+import static io.trino.spi.StandardErrorCode.TOO_MANY_REQUESTS_FAILED;
+import static io.trino.spi.StandardErrorCode.USER_CANCELED;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ExceptionsTest
```

Comment:
Please add `final` and rename to `TestExceptions`. 

https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/develop/tests.md#conventions-and-recommendations
> * Test classes should be defined as package-private and final.
> * Test class names must start with Test, for example TestExample